src/Slider/Slider.js

```diff
@@ -999,6 +999,12 @@ module.exports = kind(
 		this.set('accessibilityRole', !this.enableJumpIncrement ? 'spinbutton' : null);
 		this.set('accessibilityLive', null);
 		this.set('accessibilityHint', null);
+		this.setAriaAttribute('aria-valuetext', null);
+		this.setAriaAttribute('aria-valuenow', null);
+		if (this.$.slider) {
```

Comment:
For consistency, unless you had a specific reason otherwise let's check against `enableJumpIncrement` as we've done above.